Add default build action for 'zip' file types.

oracleguy opened the pull request:
Make zip files have the default build action of 'content'. Make sure when a new file is added that all code paths set the default build action.

EWSoftware commented:
Adding a zip file as content is fine but your second change would introduce a bug.  At that point, it's constructing a FileItem from an *existing* project element which already has a build action.  It sets the existing build action in the ProjectElement constructor that is called to create the parameter passed to the FileItem constructor.  Your change would overwrite that build action.

For example, you add a zip file to the project, change its BuildAction to None to exclude it for a while, then add it to the project again because you selected it by accident.  Now you've just changed the build action back to Content which is not what was intended but you may not notice.  So, that change needs to be reverted.

oracleguy commented:
Ok, I didn't realize that. I added that there because without it, when I added the zip the build action was set to "None". Maybe because I had added it initially to the project and then removed it via the "Exclude from project" option?

EWSoftware commented:
Merged the change to add ZIP files as content items.  Omitted the other change as noted above.
